ui/app/src/App.tsx

```diff
@@ -93,14 +99,50 @@ function App() {
       >
         <CreateStream
           streamManagerAddress={config.streamManagerAddress as `0x${string}`}
-          amountPerSecond={100000000000000}
-          registerStream={(s: Stream) => console.log(s)}
+          amountPerSecond={100}
+          registerStream={(s: Stream) => {
+            console.log(s);
+            setStreamId(s.streamId);
+          }}
```

Comment:
This gets easier too
```suggestion
          registerStream={setStream}
```